app/code/core/Mage/Sales/Model/Order/Invoice/Api.php

```diff
@@ -98,6 +98,9 @@ public function info($invoiceIncrementId)
 
         $result = $this->_getAttributes($invoice, 'invoice');
         $result['order_increment_id'] = $invoice->getOrderIncrementId();
+        $result['order_created_at'] = $invoice->getOrder()->getCreatedAt();
+        $result['billing_firstname'] = $invoice->getBillingAddress()->getFirstname();
```

Comment:
could we also add the middlename?

Reply:
Hey, I added these attributes to conform with the official M1 documentation which doesn't include middlename.

Happy to see you became a collaborator btw ;)

Reply:
Thanks @elidrissidev!

Regarding the API, if the middlename is an available field I'd add it anyway, I had to do the same "fix" on another part of the API cause the middlename was used in the database and not returned by the APIs ;-)

Reply:
Done. I will keep looking for other inconsistencies between the docs and the api and open following PRs.